RESOURCES/FEATURE_FLAGS.md

```diff
@@ -57,9 +62,26 @@ These features flags are **safe for production** and have been tested.
 - SQL_VALIDATORS_BY_ENGINE [(docs)](https://superset.apache.org/docs/installation/sql-templating)
 - SQLLAB_BACKEND_PERSISTENCE
 - THUMBNAILS [(docs)](https://superset.apache.org/docs/installation/cache)
+- ENABLE_EXPLORE_DRAG_AND_DROP
+- ENABLE_DND_WITH_CLICK_UX
+EMBEDDABLE_CHARTS
```

Comment:
a missing hyphen here makes ENABLE_DND_WITH_CLICK_UX and EMBEDDABLE_CHARTS appear on same line

Reply:
Good catch, thanks! Fixed!